packages/mobile/src/send/SendConfirmation.tsx

```diff
@@ -47,6 +67,7 @@ interface StateProps {
   confirmationInput: ConfirmationInput
   addressValidationType: AddressValidationType
   validatedRecipientAddress?: string
+  addressJustValidated?: true
```

Comment:
Shouldn't this be `boolean`?

Reply:
Could be. There's no use case for `false` but you aren't the first person to ask this so I will use `boolean` from now on to avoid confusion